core/src/main/java/com/netflix/conductor/core/config/Configuration.java

```diff
@@ -317,6 +325,28 @@ default String getElasticSearchDocumentTypeOverride() {
             ELASTIC_SEARCH_DOCUMENT_TYPE_OVERRIDE_DEFAULT_VALUE);
     }
 
+    /**
+     * @return The time to live in seconds for workflow archiving module. Currently, only RedisExecutionDAO supports it.
+     */
+    default int getWorkflowArchivalTTL() {
+        return getIntProperty(WORKFLOW_ARCHIVAL_TTL_SECS_PROPERTY_NAME, WORKFLOW_ARCHIVAL_TTL_SECS_DEFAULT_VALUE);
+    }
+
+    /**
+     * @return the time to delay the archival of workflow
+     */
+    default int getWorkflowArchivalDelay() {
```

Comment:
Consider using `getAsyncUpdateDelay` directly over this indirection to reduce confusion.